openc3-init/plugins/packages/openc3-tool-common/src/components/FileOpenSaveDialog.vue

```diff
@@ -18,7 +18,7 @@
 -->
 
 <template>
-  <v-dialog v-model="show" width="600">
+  <v-dialog v-model="show" width="600" @keydown.enter="success()">
```

Comment:
Once you select a file you can simply hit return. I've wanted to do this in the file dialog for a while now.